File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyBaronKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.razersdk.animation.Color;

import static com.bonepl.chromaleague.hud.PredefinedKeySets.BLACKWIDOW_FUNCTIONAL;

public class EnemyBaronKillAnimation extends StaticPartialBlinkingAnimation {
    public EnemyBaronKillAnimation() {
        super(BLACKWIDOW_FUNCTIONAL, 8, Color.PURPLE);
    }
}

File: src/hud/animations/EnemyBaronKillAnimation.ts
import {StaticColor} from '../../razer-sdk/color/StaticColor.js';
import {StaticPartialBlinkingAnimation} from './StaticPartialBlinkingAnimation.js';

export class EnemyBaronKillAnimation extends StaticPartialBlinkingAnimation {
    constructor() {
        super(StaticColor.PURPLE);
    }
}
